### PR TITLE
Update lockfile, fix test reporting, throwOnError on `yarn start` test page

### PR DESCRIPTION
**Pull request by ylemkimon**
Minor changes 😄

### CODE REVIEW

File: static/main.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ function init() {
     input.addEventListener("input", reprocess, false);
     permalink.addEventListener("click", setSearch);
 
-    const options = {displayMode: true, throwOnError: false};
+    const options = {displayMode: true, throwOnError: true};
```
Comment (kevinbarabash) [inline static/main.js]: This feels like a breaking change.  What's the rationale for this change?

File: test/helpers.js [hunk 2]
```diff
@@ -163,15 +163,15 @@ export const expectEquivalent = (actual, expected, settings, mode, expand) => {
...
+                `${printExpected(r(expected))} are equivalent`
             : () => {
                 const diffString = diff(expectedTree, actualTree, {
                     expand,
                 });
 
-                return `${mode.Verb} trees of ${printReceived(actual)} and ` +
-                `${printExpected(expected)} are not equivalent` +
+                return `${mode.Verb} trees of ${printReceived(r(actual))} and ` +
+                `${printExpected(r(expected))} are not equivalent` +
```
Comment (kevinbarabash) [inline test/helpers.js]: These changes look like they're improving the output of our test output, nice!

File: static/main.js (hunk 1 above)
Comment (ylemkimon) [reply static/main.js]: @ylemkimon As `yarn start` is for the debugging, I think it's better to throw on error and debug on the developer console.
